src/SDKs/Search/DataPlane/Microsoft.Azure.Search/Fix-GeneratedCode.ps1

```diff
@@ -6,6 +6,12 @@ function Replace-InFile($filePath, $oldText, $newText)
 
 $generatedFolder = ".\GeneratedSearchIndex"
 
+# Delete any extra files generated for types that are shared between SearchServiceClient and SearchIndexClient.
```

Comment:
oooohhhh you literally generate these separately? I somehow assumed this was a workaround for composite Swagger. But yeah, the fact that you had two instead of one combined generate script should've made me wonder 😄